techniques/failures/F109.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+ 
+ <head><title>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</title>
+   
+ <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></head><body><h1>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</h1><section class="meta"><p class="id">ID: F109</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
+      <p>All technologies that require authentication.</p>
+   </section><section id="description"><h2>Description</h2>
+  <p>Requiring users to authenticate by entering a password in a different format from which it was originally created is a failure to meet Success Criteria 3.3.8 and 3.3.9 (unless alternative authenticaton methods are available). Requiring users to enter individual characters of a single password across multiple fields prevents users pasting credentials from a password manager or local copy of the password. This means users cannot avoid transcription, resulting in a <a href="../../understanding/22/accessible-authentication.html#dfn-cognitive-function-test">cognitive function test</a>. This applies irrespective of whether users are required to enter all characters in the password, or a subset thereof.</p>
```

Comment:
> prevents users pasting

suggest changing to

> prevents users from pasting

---

> of whether users are required to enter all characters in the password, or a subset thereof

maybe make it a bit simpler/more readable along lines of

> of whether or not users are required to enter all characters in the password, or just a subset